README.md

```diff
@@ -1,2 +1,36 @@
-# Portfolio-setup-and-mobile-first
-this project is making of Portfolio: setup and mobile first 
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# My Portfolio
+
+> My Microverse portfolio
+
+
+## Built With
+
+- HTML/CSS
+- Figma
+
+## Authors
+
+👤 **Author1**
```

Comment:
Kindly, I suggest you put your name in this part

Reply:
@mgmediaweb 
Thanks for your good and valuable feedback.